src/python/pants/backend/python/lint/docformatter/subsystem.py

```diff
@@ -38,6 +44,26 @@ def register_options(cls, register):
                 f'`--{cls.options_scope}-args="--wrap-summaries=100 --pre-summary-newline"`.'
             ),
         )
+        register(
+            "--experimental-lockfile",
+            type=str,
+            default="<none>",
+            advanced=True,
+            help=(
+                "Path to a lockfile used for installing the tool.\n\n"
+                "Set to the string '<default>' to use a lockfile provided by "
+                "Pants, so long as you have not changed the `--version`, `--extra-requirements`, "
+                "and `--interpreter-constraints` options. See {} for the default lockfile "
```

Comment:
This will be a GitHub URL once I add a util to generate that in a follow up.

Reply:
Another option would be to literally load the resource as the default value of a field. That would allow you to both set it and override it in the option, without putting it on disk...? Not sure if that's actually great, but.

Reply:
> and override it in the option

Yeah I thought about that, but didn't like it from an ergonomics perspective:
- these files can be huge. No way you'll feasibly set it in CLI or env var, and setting in `pants.toml` is clunky.
- Pollutes `./pants help` because the value can be so big.

Instead, I think John's idea of embedding a resource is a good way to get it To Just Work by default, while still being flexible to overriding.